.github/workflows/generate_go.yaml

```
@@ -0,0 +1,24 @@
+name: generate-go-source
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+
+      - name: Generate sources
+        run: |
+          PATH=/home/runner/go/bin:${PATH}
+          make clean all
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Generate Golang source
```

No newline at end of file
```suggestion
          commit_message: Generate Golang source

```